### ISSUE TITLE
Seeing  people that are not on my local network

### ISSUE THREAD

**Issue by capttony** [RobinLinus/snapdrop#548]
I do not want my application to look like snapdrop.app. wher you see all of these people not on your network. My understanding is if you are running snapdrop externally and you only want to see the people that are on your network that you need to make sure your reverse proxy goes right to the node server on port 3000, NOT to the webserver on port 8081. I do not know how to do this. Can someone please advise what I need to change and where do I change. I am using clouldflare tunnel as my reverse proxy

**Comment by mwy001** [comment]
You have to pass the "real IP address" of peers through all of the proxies to the node server. How to achieve this depends on the reverse proxies you are using. For example on nginx it is:
    proxy_set_header X-Forwarded-for $proxy_add_x_forwarded_for;

**Comment by capttony** [comment]
Thank you for your reply. I tried adding that line here with no change:
app/www/docker/nginx# nano default.conf

location /server {
        proxy_connect_timeout 300;
        proxy_pass http://localhost:3000;
        proxy_set_header Connection "upgrade";
        proxy_set_header Upgrade $http_upgrade;
        proxy_set_header X-Forwarded-for $remote_addr;
        proxy_set_header X-Forwarded-for $proxy_add_x_forwarded_for;

Is this the correct path?

**Comment by mwy001** [comment]
I think there is no need to update the Nginx conf of snapdrop code for this to work. Only need to update the reverse proxy conf placed in front of snapdrop deployment.

You mentioned cloudflare, does the following help?
https://developers.cloudflare.com/fundamentals/get-started/reference/http-request-headers/#x-forwarded-for

**Comment by capttony** [comment]
Thank you for the link on the cloudflare details. I played with some of the settings but did nothing changed. My knowledge is limited as I am a beginner. Before I was using Cloudflare tunnel I was using just the Nginx proxy manager with exposed ports and was getting the same result. The odd thing is I am also able to send files when not on the same network. I thought this was not suppose to work this way but it does. I went back and looked through the install instructions [ https://hub.docker.com/r/linuxserver/snapdrop ] and they say "If you intend to expose Snapdrop to the internet, edit /config/nginx/site-confs/default.conf and uncomment the real_ip settings" I have looked in this default config folder and there is no real ip settings to uncomment. Is there something I'm missing here and could this be causing my problem?
